wslpy/system.py

```diff
@@ -78,4 +78,67 @@ def shellEnvVar(input):
 
 
 def registry(input, key):
-    raise NotImplementedError
\ No newline at end of file
+
+    #DUMMY CODE BEGINS (i am only using this to illustrate something)
+    cmd = u"reg.exe query \""+input+"\" /v \""+key+u"\" 2>&1"
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+    routput, err = p.communicate()
+    #DUMMY CODE ENDS 
+
+    # Given a valid registry path, retrieves the value of an entry in the registry.
+    # Eg: registry("HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment","OS") returns "WINDOWS_NT"
```

Comment:
Did I understand the code right? Is this what the library is designed to take in?

Reply:
Yep, this is right. Also if possible, the optimal goal is to return its type too:
![image](https://user-images.githubusercontent.com/15316889/89114583-aa04df00-d4b0-11ea-97e6-a2985a9895c2.png)

Reply:
Ah, shall we return this as an array or a class object?

Reply:
Added in the new commit, see https://github.com/wslutilities/wslpy/pull/13/commits/6fc47610d174f222973387da362e454b654ac5f0